### PR TITLE
Fix Sidebar toggle bug

### LINKED ISSUE #128: [BUG] Sidebar
### Describe the bug

When we click on the sidebar-nav item, it automatically closes the sidebar and opens the new page.
whereas when we click on the dropdown nav-items, it doen;t close the sidebar automatically
under Learn navitem:
- navitem > learn
- navitem > learn > prep

drawing
### To Reproduce

- navitem > events:
  - works.
- navitem > learn > prep > quiz:
  - does not work.

### Expected Behavior

When we click on:
- navitem > learn > prep > quiz:

it should work the same as events and blogs.

### Screenshot/ Video

screenshot: 
drawing

video:

https://user-images.githubusercontent.com/44284877/194047248-01af7233-7c4f-4140-9415-c4b791eef504.mp4

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by abdullmustyy**
## Fixes Issue

Closes #128 

## Screenshots

https://user-images.githubusercontent.com/87391935/195264200-c0791e7a-a311-4d54-8843-d4e3b5be62d3.mp4

## Code of Conduct
- [x] By submitting this pull request, I confirm I've read and complied with the [CoC](https://github.com/thecyberworld/Support/blob/main/CODE_OF_CONDUCT.md) 🖖

## Check List (Check all the applicable boxes) 
- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

---

You can also join our [Discord](https://discord.gg/QHBPq6xP5p) community. 
Feel free to check out other cool repositories of the [Thecyberworld](https://github.com/thecyberworld). 
Join the Thecyberworld GitHub Organisation by raising an [issue](https://github.com/thecyberworld/Support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) (you will be sent an invitation).

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "thecyberhub.org",
-      "version": "0.15.2",
+      "version": "0.15.9",
```
Comment (kabir0x23) [inline package-lock.json:9]:
```suggestion
      "version": "0.15.2",
```

File: package-lock.json [hunk 2]
```diff
@@ -7033,4 +7033,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (kabir0x23) [inline package-lock.json:7036]:
```suggestion
}
```

**Review by kabir0x23 — CHANGES_REQUESTED** [review]

**Comment by abdullmustyy** [comment]
Is there something I need to do?

**Comment by abdullmustyy** [comment]
> Is there something I need to do?

@kabir0x23

**Comment by kabir0x23** [comment]
> Is there something I need to do?

nope, same package.json issue, i have resolved it

**Comment by abdullmustyy** [comment]
> > Is there something I need to do?
> 
> nope, same package.json issue, i have resolved it

Oh, alright.
Thank you.

**Comment by kabir0x23** [comment]
> > > Is there something I need to do?
> > 
> > 
> > nope, same package.json issue, i have resolved it
> 
> Oh, alright. Thank you.

Heard about hacksquad?

**Comment by kabir0x23** [comment]
> eard about hacksquad? @abdullmustyy
> 
> that thing is same as hacktober but can win more swags, with same contributions'
> 
> have to join with the team, top60 will get the swags

register link
https://www.hacksquad.dev/friend/a2FiaXJqMjMxMEBnbWFpbC5jb20=

will send you the join link after this

**Comment by abdullmustyy** [comment]
> eard about hacksquad? @abdullmustyy
> 
> that thing is same as hacktober but can win more swags, with same contributions'
> 
> have to join with the team, top60 will get the swags

Mnnn, that's great. 
I've never heard of it.

**Comment by abdullmustyy** [comment]
> > eard about hacksquad? @abdullmustyy
> > that thing is same as hacktober but can win more swags, with same contributions'
> > have to join with the team, top60 will get the swags
> 
> register link https://www.hacksquad.dev/friend/a2FiaXJqMjMxMEBnbWFpbC5jb20=
> 
> will send you the join link after this

Alright, I'll register now.

**Comment by kabir0x23** [comment]
> > > eard about hacksquad? @abdullmustyy
> > > that thing is same as hacktober but can win more swags, with same contributions'
> > > have to join with the team, top60 will get the swags
> > 
> > 
> > register link https://www.hacksquad.dev/friend/a2FiaXJqMjMxMEBnbWFpbC5jb20=
> > will send you the join link after this
> 
> Alright, I'll register now.

invite link
https://www.hacksquad.dev/invite/cl9589xkr000ml409vu53777r

**Comment by abdullmustyy** [comment]
I'm in! 
What next?

**Comment by kabir0x23** [comment]
> I'm in! What next?

awesome,

just simple, same as hack
if you submitting a pr, it will reflects in both sites (same pr, double benifits)

**Comment by kabir0x23** [comment]
> > I'm in! What next?
> 
> awesome,
> 
> just simple, same as hack if you submitting a pr, it will reflects in both sites (same pr, double benifits)

it will take hrs to reflect, once it reflects, we will be in top 30

**Comment by abdullmustyy** [comment]
> > > I'm in! What next?
> > 
> > 
> > awesome,
> > just simple, same as hack if you submitting a pr, it will reflects in both sites (same pr, double benifits)
> 
> it will take hrs to reflect, once it reflects, we will be in top 30

Cooool!
So I basically just have to keep contributing to projects with Hactoberfest labels, yeah?

**Comment by kabir0x23** [comment]
> Cooool! So I basically just have to keep contributing to projects with Hactoberfest labels, yeah?
https://www.hacksquad.dev/#qa
yes, but with this we can contribute to - Any public repository you want! Please make sure not to spam! We check 🤫

**Comment by abdullmustyy** [comment]
> > Cooool! So I basically just have to keep contributing to projects with Hactoberfest labels, yeah?
> > https://www.hacksquad.dev/#qa
> > yes, but with this we can contribute to - Any public repository you want! Please make sure not to spam! We check 🤫

Lol, I actually don't know what it means to spam. 😅

**Comment by kabir0x23** [comment]
> Lol, I actually don't know what it means to spam. 😅

spam: just changing random stuff: like README.md -> readme.md
adding random things in the files and send the pr 

this is because:
once in 2020, i person made a video free t-shirts , do this to get

so 1000s of people were start doing this thing,
and start seding invalid and spammy prs
that time, just have to send prs, prs should be open (merge was not reqiure) and should not be decliend 

now you can get what and why spam is mentioned

**Pull request merged**